doc/holochain_101/src/zome/intro_to_hdk.md

```diff
@@ -36,7 +36,7 @@ Once Holochain stabilizes beyond the 0.0.x version numbers, it will be published
 
 If you wanted to lock the HDK at a specific version, you could adjust the HDK dependency like this:
 ```toml
-hdk = { git = "https://github.com/holochain/holochain-rust", tag = "holochain-cli-v0.0.3" }
+hdk = { git = "https://github.com/holochain/holochain-rust", tag = "holochain-cli-v0.0.4" }
```

Comment:
I think this isn't a valid tag name anymore, so we should change the format of this